cloud

```diff
@@ -1974,6 +1980,9 @@ function usage_deploy() {
     echo "    --log-level=NAME"
     echo "              Specify Python log level NAME for Cloud Functions."
     echo "              Default is INFO."
+    echo "    --optimize=PYTHONOPTIMIZE"
+    echo "              Specify a value for PYTHONOPTIMIZE to be added to the environment"
+    echo "              Default is Zero."
```

Comment:
Now, since you're mentioning that we're expecting a value for `PYTHONOPTIMIZE` explicitly (and that's good), let's write `Default is empty string`. That would also match the tests I described yesterday.

Reply:
Oh! Okay.